cfme/infrastructure/virtual_machines.py

```diff
@@ -265,7 +265,10 @@ class TemplatesToolbar(InfraGenericDetailsToolbar):
 
     @property
     def is_displayed(self):
-        expected_name = self.context['object'].name
+        expected_titles = [
+            'VM and Instance "{}"'.format(self.context["object"].name),
```

Comment:
Discussed offline a bit - this is a crossover of VM/Template that we need to split up a bit to keep the navigation and view detection accurate and reliable.

We either need two views (separate class for template details view), or is_displayed need to detect what type of contextual object is there, and set the title respective to the class.

That will drive whether or not the navigation destination needs to be changed. If we have a separate view for Template, then we'll need either two navigation destination classes or a `VIEW` property that detects the object type and returns the correct view matching the class.

Reply:
@mshriver I selected the first option.